### PR TITLE
Add a note concerning replying in a thread

### PR DESCRIPTION
**Pull request by Startouf**
For https://github.com/slack-ruby/slack-ruby-bot/issues/131

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,6 +106,21 @@ end
 
 Operator match data includes `match['operator']` and `match['expression']`. The `bot` match always checks against the `SlackRubyBot::Config.user` setting.
 
+You can reply to a message in a thread (or keep sending to the current thread) :
+
+```ruby
+command('reply in thread') do |client, data, match|
```
Comment (kstole) [inline README.md]: Can we remove the parentheses for consistency with the rest of the documentation

File: README.md [hunk 2]
```diff
@@ -106,6 +106,21 @@ end
...
+  client.say(
+    channel: data.channel,
+    text: "let's avoid spamming everyone, I will tell you what you need in this thread",
+    thread_ts: (data.thread_ts || data.ts)
```
Comment (kstole) [inline README.md]: Parentheses not needed here either

File: README.md [hunk 3]
```diff
@@ -106,6 +106,21 @@ end
 
 Operator match data includes `match['operator']` and `match['expression']`. The `bot` match always checks against the `SlackRubyBot::Config.user` setting.
 
+You can reply to a message in a thread (or keep sending to the current thread) :
```
Comment (kstole) [inline README.md]: I think this should have it's own section, like `### Threaded Messages`

File: README.md [hunk 4]
```diff
@@ -106,6 +106,23 @@ end
 
 Operator match data includes `match['operator']` and `match['expression']`. The `bot` match always checks against the `SlackRubyBot::Config.user` setting.
 
+### Threaded Messages
+
+You can reply to a message in a thread (or keep sending to the current thread) :
```
Comment (dblock) [inline README.md]: Nitpick: why a space before `:`?

File: README.md [hunk 5]
```diff
@@ -106,6 +106,23 @@ end
...
+
+```ruby
+command 'reply in thread' do |client, data, match|
+  client.say(
+    channel: data.channel,
+    text: "let's avoid spamming everyone, I will tell you what you need in this thread",
+    thread_ts: data.thread_ts || data.ts
```
Comment (dblock) [inline README.md]: Add a comment to what these are? Maybe a link to docs?

Comment (Startouf) [reply README.md]: Hello sorry for the long wait. I'm a French guy so sometimes I forget there is no space before semicolons in english ^^"

I have included extra information and links regarding `thread_ts` and `ts`.

Sorry for stalling this PR for so long

**Comment by dblock** [comment]
Closed via https://github.com/slack-ruby/slack-ruby-bot/commit/162b9e04dffb52fb45c2ff4118e80c009cbf404f

**Pull request closed without merging**